src/web/components/ApiKeyManagement/keyEditDialog.spec.tsx

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable camelcase */
+import { composeStories } from '@storybook/testing-react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+
+import KeyEditDialog, {  } from './KeyEditDialog';
+import * as stories from './KeyEditDialog.stories';
+
+const { KeyWithRolesParticipantIsntAllowed, MultipleRoles } = composeStories(stories);
+
+describe('Key edit dialog', () => {
+  it('has the correct title', () => {
```

Comment:
I do wonder how much value this test adds - it seems more like a basic "did it successfully render" test which might be better asserted by checking for the presence of a checkbox or a save/cancel button? Then it won't be tightly coupled with the title which doesn't really influence the behaviour and may be more subject to change? What do you think?